nym-api/src/support/http/mod.rs

```diff
@@ -154,3 +151,50 @@ fn setup_cors() -> Result<Cors> {
 
     Ok(cors)
 }
+
+#[allow(dead_code)]
+// TODO dz Cannot combine `Access-Control-Allow-Credentials: true` with `Access-Control-Allow-Headers: *`
```

Comment:
imo just remove the allow credentials then because we really don't care about those. we just needed the headers; the reason I had the credentials on my branch is that it was the default behaviour from rocket's cors setup

Reply:
done